contracts/security-token/tests/InvestorInteractionContract.sol

```diff
@@ -69,7 +69,7 @@ contract InvestorInteractionContract is BogusAnnouncement, CheckpointToken, ERC8
     }
 
     if (options[_to] != 0) {
-      require(KYC.isWhitelisted(msg.sender));
+      require(KYC.getFlag(msg.sender, 0));
```

Comment:
Lets use an enumeration instead of numbers like 0 to indicate what this means. For eg in python it would like:
```
class KYCMode(enum.Enum):
   user_cleared_kyc = 0b01  # binary representation
   admin_allowed_to_transfer = 0b10
```
and then KYCMode.user_cleared_kyc.value

Reply:
Yes, this is a good idea :+1: